hr_infraction/__init__.py

```diff
@@ -1,23 +1,7 @@
-# -*- coding:utf-8 -*-
-#
-#
-#    Copyright (C) 2013 Michael Telahun Makonnen <mmakonnen@gmail.com>.
-#    All Rights Reserved.
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as published
-#    by the Free Software Foundation, either version 3 of the License, or
-#    (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#
+# -*- coding: utf-8 -*-
+# © 2013 Michael Telahun Makonnen <mmakonnen@gmail.com>
+# © 2016 OpenSynergy Indonesia
+# @author Andhitia Rama
```

Comment:
Please remove the author here, you already are in the contribuitor tab of the module.